src/commands/account/info.ts

```diff
@@ -87,10 +89,29 @@ export default class extends ProfileCommand {
                 text += 'Importance:\t' + accountInfo.importance.compact() + '\n';
                 text += 'at height:\t' + accountInfo.importanceHeight.compact() + '\n\n';
                 text += 'Mosaics' + '\n';
-                accountData.mosaics.map((mosaic: MosaicAmountView) => {
+                accountData.mosaics.map((mosaic: MosaicAmountView, index: any) => {
                     text += mosaic.fullName() + ':\t' + mosaic.relativeAmount() + '\n';
+                    text += 'expiration height:\t' + '{height' + index + '}' + '\n\n';
+                    mosaicsId.push(new MosaicId(mosaic.fullName()));
+                    mosaicsHexId.push(mosaic.fullName());
                 });
-                console.log(text);
+                mosaicHttp.getMosaics(mosaicsId)
+                    .subscribe((data) => {
```

Comment:
Nesting subscribes is not considered a best practice. Please, see how can you resolve it using [mergeMap](https://stackoverflow.com/questions/55416011/how-can-i-avoid-multiple-nested-subscriptions-using-rxjs-operators) or other RxJS operators.